src/main/java/com/example/Bloggy/model/BlogPost.java

```diff
@@ -0,0 +1,57 @@
+package com.example.Bloggy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class BlogPost {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    private String author;
+    private String tags;
+    private String title;
+    private String content;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
```

Comment:
do we need to set ID? or should it be set up by database?

Reply:
SQL sets ID as intended. Setter removed.